SwiftUIDZ/SwiftUIDZ/MainTabView.swift

```diff
@@ -0,0 +1,57 @@
+// MainTabView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Таббар
+struct MainTabView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let navBarHeight: CGFloat = 30
+        static let yellowColorName = "pastelYellow"
```

Comment:
```suggestion
        static let yellowColorName = "pastelYellow"
```
ты ж выносила цвета, вынеси этот тоже